deploy/template.yaml

```diff
@@ -888,336 +879,6 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref InitialiseIpvSessionFunctionLogGroup
 
-  RetrieveCriOauthAccessTokenFunction:
```

Comment:
Hard question alert - do you think there's any risk that we could tear down the old lambdas while users are still using the old CRI return?

I don't immediately know the answer... If we're worried we can deploy the API GW part of the change first, but it's a bit faffy. If it does go wrong it should only be a small number of people, but still :/

Reply:
Merging the open api spec should be doable, I'll create a separate PR for it